data/mods/gen1/moves.ts

```diff
@@ -604,7 +604,7 @@ export const Moves: {[k: string]: ModdedMoveData} = {
 		damageCallback(pokemon) {
 			const psywaveDamage = (this.random(0, this.trunc(1.5 * pokemon.level)));
 			if (psywaveDamage <= 0) {
-				this.hint("Desync Clause activated.");
+				this.hint("Desync Clause Mod activated.");
```

Comment:
```suggestion
				this.hint("Desync Clause Mod activated!");
```

If you want to be consistent with my commit (and with Pokémon's general rule of putting exclamation marks on most everything), this is an exclamation mark. ;)